plasmapy/utils/datatype_factory_base.py

```diff
@@ -177,21 +169,17 @@ def register(self, WidgetType, validation_function=None, is_default=False):
             for vfunc_str in self.validation_functions:
                 if hasattr(WidgetType, vfunc_str):
                     vfunc = getattr(WidgetType, vfunc_str)
-
-                    # check if classmethod: stackoverflow #19227724
```

Comment:
Are you okay with this comment being removed?  I'm only pointing it out since, I'm assuming, Sourcery is the one automating the removal and it likely needs a human verification.

Reply:
I manually refactored this by renaming the intermediate variable to `is_not_classmethod` to make the comment less needed.  I could put the stackoverflow link back in if you'd prefer, but I'm okay with removing the comment.

Reply:
I'm fine either way.  Just wanted to make sure human eyes saw this and confirmed it.